vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/gke-install.md

```diff
@@ -0,0 +1,237 @@
+1. Login to your Google Cloud account: <https://cloud.google.com/>
```

Comment:
I think we lost some content on the new version. Usually there's an Overview, Prerequisites, etc. That could probably be lifted from the previous version.

Reply:
added